pkg/field/defaults.go

```
@@ -45,9 +62,38 @@
+		WithPersistent(true), WithExportTarget(ExportTargetOps))
+	otelLoggingDisabled = BoolField(OtelLoggingDisabledFieldName, WithDescription("Disable OpenTelemetry logging"), WithDefaultValue(false),
+		WithPersistent(true), WithExportTarget(ExportTargetOps))
+	otelTracingEndpoint = StringField(OtelTracingEndpointFieldName, WithDescription("The endpoint of the OpenTelemetry collector to send tracing data to (overrides otel-collector-endpoint for tracing)"),
+		WithPersistent(true), WithExportTarget(ExportTargetOps))
+	otelTracingEndpointTlsCertPath = StringField(OtelTracingEndpointTLSCertPathFieldName, WithDescription("Path to a file containing a PEM-encoded certificate to use as a CA for TLS connections to the OpenTelemetry tracing collector"),
+		WithPersistent(true), WithExportTarget(ExportTargetOps))
+	otelTracingEndpointTlsCert = StringField(OtelTracingEndpointTLSCertFieldName, WithDescription("A PEM-encoded certificate to use as a CA for TLS connections to the OpenTelemetry tracing collector"),
+		WithPersistent(true), WithExportTarget(ExportTargetOps))
+	otelTracingEndpointTlsInsecure = BoolField(OtelTracingEndpointTLSInsecureFieldName, WithDescription("Allow insecure connections to the OpenTelemetry tracing collector"),
+		WithPersistent(true), WithExportTarget(ExportTargetOps))
+	otelLoggingEndpoint = StringField(OtelLoggingEndpointFieldName, WithDescription("The endpoint of the OpenTelemetry collector to send logging data to (overrides otel-collector-endpoint for logging)"),
```

golangci-lint (1.64.8)

[error] line-length-limit: line is 203 characters, out of limit 200

(revive)